airflow/www/views.py

```diff
@@ -681,9 +682,17 @@ def dag_details(self, session=None):
             .group_by(TI.state)\
             .all()
 
+        active_runs = models.DagRun.find(
+            dag_id=dag.dag_id,
+            state=State.RUNNING,
```

Comment:
thanks. I go back and check https://github.com/apache/airflow/blob/master/airflow/models/__init__.py#L3560 which we define active_run to only running state. Initially, I thought scheduled would be one of them as well.